geode-for-redis/src/distributedTest/java/org/apache/geode/redis/internal/commands/executor/list/LPushDUnitTest.java

```diff
@@ -114,7 +115,8 @@ public void givenBucketsMovedDuringLPush_elementsAreAddedAtomically()
 
     for (String key : keys) {
       long length = jedis.llen(key);
-      assertThat(length).isGreaterThanOrEqualTo(MINIMUM_ITERATIONS * 2 * PUSH_LIST_SIZE);
+      assertThat(length).isCloseTo(ITERATION_COUNT * 2 * PUSH_LIST_SIZE, within(6L));
+      assertThat(length % 3).isEqualTo(0);
```

Comment:
I think this assertion can be exact and shouldn't need the offset.

Reply:
I have seen failures, but that was an earlier version. Ran a bunch of these without failures, so I'm making it exact.